packages/sra-types/README.md

```diff
@@ -0,0 +1,69 @@
+## @0xproject/types
```

Comment:
Should be `sra-types`, but also why did you decide not to just put these in types? Or just depend on connect in your package?

Reply:
You're right. Moved everything into types